examples/kubernetes-istio/cilium-kube-inject.sed

```diff
@@ -0,0 +1,13 @@
+# Inject Cilium's /var/run/cilium directory into each Istio sidecar proxy to
+# allow Cilium's Envoy filters to communicate with the Cilium agent.
+#
+# volumeMounts:
+# - mountPath: /var/run/cilium
```

Comment:
We'll want to share a subdirectory in the future so we don't have to expose the entire state directory with each sidecar.

Reply:
Is it possible to mount the .sock directly? I've seen it done with docker but I don't know if volumeMounts/volumes allows it.

Reply:
Maybe. I’ll look into that. I have 2 sockets to mount. I went with the simplest solution for now.